merlin/helpers.py

```diff
@@ -65,3 +65,22 @@ def clean_whitespace(text):
     """
 
     return ' '.join(text.split())
+
+
+def make_snippet(name, desc):
+    """
+    Converts name and desc provided by merlin autocomplete to
+    a Sublime Text snippet
+    """
+    snippet = name
+
+    # TODO: Need a better way of splitting params, "->"
```

Comment:
This is an important todo which I haven't yet addressed yet either, but it's pretty complex. It's like we need a fully parsed tree. 

Turns out the resulting auto-complete snips just suck without it when a parameter is a function (which is like all the time in Ocaml)